.test-infra/mock-apis/src/main/go/internal/metric/doc.go

```diff
@@ -0,0 +1,17 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package metric supports monitoring.
+package metric
```

Comment:
Style note: If the documentation is small, and the package is small (file wise), it's better to just put it on top of one ot the non test files. A separate doc file works best when the package documentation is long, or doesn't naturally fit into one of the existing files.

Usually though in that case there'd be a file named the same as the package. If it were me, I'd rename interface.go to metric.go since it's an evident entry point to the package.

Reply:
I was anticipating it being longer in the beginning but then realized it didn't need to be. I won't change anything at this point though unless you or others feel strongly about it.